src/alexjs/alex.service.ts

```diff
@@ -23,7 +23,18 @@ export class AlexService {
       .messages;
 
     const notifications: MessageEmbed[] = [];
-    if (alexMatch.length) {
+    if (
+      alexMatch.length &&
+      !message.content.includes(
+        'discord.gg/' ||
+          'discordapp.com/invite/' ||
```

Comment:
I'm not sure if we can check with a `String.includes` for that.
In my opinion we need to check the string with a Regex or the `URL`-constructor if its a valid URL.
We could wrap that in a private function and specify the allowed "string-snippets" there.
What do you think about that @eddiejaoude

Reply:
Yep, I think a regex would be better. Becca bot has a regex we could use https://github.com/BeccaLyria/discord-bot/blob/main/src/listeners/linksListener.ts#L46

Reply:
Changed now. Let me know if it's fine. Thanks. 🙂